src/System.Windows.Forms/tests/UnitTests/AccessibleObjects/ControlAccessibleObjectTests.cs

```diff
@@ -0,0 +1,120 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using Xunit;
+
+namespace System.Windows.Forms.Tests.AccessibleObjects
+{
+    public class ControlAccessibleObjectTests
```

Comment:
Should it be the following?
```suggestion
    public class Control_AccessibleObjectTests
```
...and filename `Control.AccessibleObjectTests.cs`?

Reply:
This source base is not using the new convention yet, are you sure we want to enforce this convention in this branch?

Reply:
We started using this convention as agreed with the Akvelon team a month ago. We can revisit it once all major outstanding PRs are resolved to reduce contention.